cmake/all.cmake

```diff
@@ -162,6 +166,11 @@ set(CATKIN_GLOBAL_LIBEXEC_DESTINATION lib)
 set(CATKIN_GLOBAL_PYTHON_DESTINATION ${PYTHON_INSTALL_DIR})
 set(CATKIN_GLOBAL_SHARE_DESTINATION share)
 
+# This was removed, but is required for Windows
```

Comment:
I have tested this code.  It is used to ensure that during initial bootstrap using catkin_make_isolated, the shared library settings are set properly.  Otherwise library creation fails due to lack of ROS_BUILD_SHARED_LIBS definition and no toplevel CMakeLists.txt since catkin_init_workspace does not exist yet.  I can change the comment to reflect this, or look for a better solution.